src/mumble/CustomElements.cpp

```diff
@@ -152,6 +153,24 @@ void ChatbarTextEdit::setDefaultText(const QString &new_default, bool force) {
 	}
 }
 
+void ChatbarTextEdit::insertFromMimeData(const QMimeData *source) {
+	if (source->hasImage()) {
+		if (g.bAllowHTML) {
+			QImage image = qvariant_cast<QImage>(source->imageData());
+
+			QString imgHtml = QLatin1String("<br />") + Log::imageToImg(image);
```

Comment:
Why do we need a leading newline?

Reply:
Nvm - Just realized that this is to move the image to another line than the `To...` stuff :bulb: